### DISCUSSION TITLE
Subscriptions in Graphiql

### DISCUSSION THREAD

**Discussion by ppbpdx** [mirumee/ariadne#1094] [Q&A]
I am upgrading my app to the latest version using the FastApi example. 

Previously I mounted the subscriptions handler like this:

```app.mount("/ws", GraphQL(subscription_schema, debug=True))```

And if I pointed a browser to localhost/ws the graphiql interface let me test out and experiment with the subscriptions. This was very useful to debug and document the subscriptions.

In the new documentation example I am mounting my subscriptions like this:
``` 
@app.websocket("/api")
async def graphql_subscriptions(
    websocket: WebSocket,
    db = Depends(get_db),
):
    # Expose database connection to the GraphQL through request's scope
    websocket.scope["db"] = db
    await graphql_app.handle_websocket(websocket)
```

However when I try to test a subscription through graphiql I get an error:

```
"message": "Your GraphiQL createFetcher is not properly configured for websocket subscriptions yet. Please provide subscriptionUrl, wsClient or legacyClient option first.",
```

Is there a good way to enable testing subscriptions through graphiql that anyone can recommend? Is this something I need to configure in the client code?

Thanks for the help!

**Comment by Princeigwe** [comment]
I am facing this same issue in Django 😭
